package.json

```diff
@@ -142,7 +142,7 @@
     "nodemailer": "^6.7.5",
     "nyc": "^15.1.0",
     "proxyquire": "^2.1.3",
-    "pubnub": "^7.0.1",
+    "pubnub": "^8.2.5",
```

Comment:
It's a BREAKING CHANGE. Please mark it so.

Reply:
ok sir